data-model/fdo-types/0.3.0/digital-specimens/examples/digital-specimen-bare.json

```diff
@@ -0,0 +1,127 @@
+  "ods:hasMaterialEntity": [],
+  "ods:hasIdentification": [
+    {
+      "@type": "dwc:Identification",
+      "dwc:verbatimIdentification": "Pitta versicolor concinna Gould, 1857",
+      "dwc:identifiedBy": "Tan D.",
+      "dwc:dateIdentified": "2021/12/01",
+      "dwc:identificationVerificationStatus": true,
+      "ods:hasTaxonIdentification": [
+        {
+          "@type": "ods:TaxonIdentification",
+          "dwc:taxonID": "6VM3W",
+          "dwc:scientificName": "Pitta versicolor Swainson, 1825",
+          "ods:scientificNameHtmlLabel": "<i>Pitta versicolor</i> Swainson, 1825",
+          "dwc:scientificNameAuthorship": "Swainson, 1825",
+          "dwc:namePublishedInYear": "1825",
+          "dwc:taxonRank": "SPECIES",
+          "dwc:kingdom": "Animalia",
+          "dwc:phylum": "Chordata",
+          "dwc:class": "Aves",
+          "dwc:order": "Passeriformes",
+          "dwc:family": "Pittidae",
+          "dwc:genus": "Pitta Vieillot, 1816",
+          "dwc:specificEpithet": "versicolor",
+          "dwc:taxonomicStatus": "ACCEPTED",
+          "dwc:nomenclaturalCode": "ICZN",
+          "dwc:genericName": "Pitta"
+        }
+      ]
+    }
+  ],
+  "ods:hasAssertion": [],
+  "ods:hasEvents": [
+    {
+      "@type": "ods:Event",
+      "dwc:organismQuantity": "1",
+      "dwc:organismQuantityType": "individuals",
+      "dwc:lifeStage": "adult",
+      "dwc:eventDate": "1880-09-01/1880-09-30",
+      "ods:Assertions": [],
+      "dcterms:Location": {
+        "dwc:stateProvince": "Alor",
+        "ods:GeoReference": {
+          "dwc:geodeticDatum": "WGS84"
+        },
+        "ods:GeologicalContext": {}
+      }
+    }
+  ],
+  "ods:hasEntityRelationship": [
+    {
+      "@type": "ods:EntityRelationship",
+      "ods:entityRelationshipType": "hasOrganisationId",
+      "ods:objectEntityIri": "https://ror.org/0566bfb96"
+    },
+    {
+      "@type": "ods:EntityRelationship",
+      "ods:entityRelationshipType": "hasSourceSystemId",
+      "ods:objectEntityIri": "https://hdl.handle.net/SANDBOX/CS1-ZKQ-NZ7"
+    },
+    {
+      "@type": "ods:EntityRelationship",
+      "ods:entityRelationshipType": "hasFdoType",
+      "ods:objectEntityIri": "https://doi.org/21.T11148/894b1e6cad57e921764e"
+    },
+    {
+      "@type": "ods:EntityRelationship",
+      "ods:entityRelationshipType": "hasPhysicalIdentifier",
+      "ods:objectEntityIri": "https://data.biodiversitydata.nl/naturalis/specimen/RMNH.AVES.121559"
+    },
+    {
+      "@type": "ods:EntityRelationship",
+      "ods:entityRelationshipType": "hasColId",
+      "ods:objectEntityIri": "https://www.catalogueoflife.org/data/taxon/6VM3W",
+      "ods:entityRelationshipDate": 1716966655568,
+      "ods:entityRelationshipCreatorName": "dissco-nusearch-service",
+      "ods:entityRelationshipCreatorId": "https://hdl.handle.net/TEST/123-123-123"
+    }
+  ],
+  "ods:hasCitation": [
+    {}
+  ],
+  "ods:hasIdentifier": [
```

Comment:
Should 'ods:normalisedPhysicalSpecimenID' be in this list as well? 
```

{
      "@type": "ods:Identifier",
      "ods:identifierType": "ods:normalisedPhysicalSpecimenID",
      "ods:identifierValue": "https://data.biodiversitydata.nl/naturalis/specimen/RMNH.AVES.121559"
    }
```

Reply:
Yes we can include these but that doesn't change the data model right? It would just be one more identifier in the hasIdentifier part